ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -60,6 +61,10 @@
       </div>
     {{/if}}
   {{else}}
-    <EmptyState @title="No mount selected" @message="Select a mount above to get started." />
+    <EmptyState
+      @title="No mount selected"
+      @message="Select a mount above to get started."
+      data-test-no-mount-selected-empty
```

Comment:
It seems like you could use the empty state component test selectors and just reuse those in your test file?